mail/postfix/Makefile

```diff
@@ -177,136 +181,90 @@ newaliases_path=/usr/bin/newaliases
 mailq_path=/usr/bin/mailq
 shlib_directory=/usr/lib/postfix
 meta_directory=/etc/postfix
+persistent_data=/usr/var
+# TODO:
+# - the postinit script should check if an external storage device is mounted
+# - if an external device is found, use that device's mount point
+# - if extroot is used, use $(persistent_data)
+data_directory=$(persistent_data)/postfix/lib
+queue_directory=$(persistent_data)/postfix/spool
+mail_spool_directory=$(persistent_data)/postfix/mail
```

Comment:
Please keep the directories as they were, i.e. $(persistent_data)/lib/postfix. I don't want to change the directories too often to avoid creating several unused directories on user devices.